### DISCUSSION TITLE
Diagram visuals

### DISCUSSION THREAD

**Discussion by RNSS** [jgraph/drawio-desktop#1672] [General]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
- The text is very close to borders
- Ability to change default font
- Ability to change default shape borders thickness 

**Describe the solution you'd like**
- Add more space in all directions
- Ability to change default font
- Ability to change default shape borders thickness 

**Describe alternatives you've considered**

**Additional context**

**Comment by davidjgraph** [comment]
This is too vague and contains multiple issues. I can convert this is to a discussion, but it's zero use to anyone in it's current form.

**Comment by RNSS** [comment]
Thank you for the response. Here is detailed issues.
- The text is very close to borders
the shape should be more dynamic and increase it size and keep minimum space between text and shape borders
<img width="438" alt="Screenshot 2024-03-24 at 3 06 09 AM" src="https://github.com/jgraph/drawio-desktop/assets/40342056/0512dad2-1b48-48a6-b6d7-892b128e9a9f">

- Ability to change default font: 
Let's take this steps as example,
1. insert rectangle shape
2. change the font to Courior New 
3. click Set As default style
4. insert circle shape, write any text -> the font is Courior New. GREAT
5. Start new Diagram
6. insert rectangle shape, write any text -> the font is Helvetica. NOOOO :(
Note: Clearly the font is attached with global Style (the tab shows when you click on any anywhere), So How I can make my custom style and make it the default?
<img width="618" alt="Screenshot 2024-03-24 at 3 59 48 AM" src="https://github.com/jgraph/drawio-desktop/assets/40342056/c1dba41d-61ae-45d2-a64a-69ca2bd10327">
<img width="523" alt="Screenshot 2024-03-24 at 3 59 20 AM" src="https://github.com/jgraph/drawio-desktop/assets/40342056/bc7aaf9d-1bc2-4847-b601-f658f2ea7f53">
<img width="244" alt="Screenshot 2024-03-24 at 3 48 41 AM" src="https://github.com/jgraph/drawio-desktop/assets/40342056/9e7d95a3-51e4-4644-a265-ac4971240ad7">

- Ability to change default shape borders thickness:
Same above It should have the ability to make this style default in any diagram created.
